sdk-wallet/user_wallet_provider.md

```diff
@@ -28,6 +31,14 @@ class UserWalletProvider:
 
     // Should not throw.
     generate_mnemonic(): Mnemonic
+
+    // Should not throw.
+    validate_mnemonic(mnemonic: Mnemonic): bool
+
+    derive_secret_key_from_mnemonic(mnemonic: Mnemonic, address_index: int, passphrase: string): ISecretKey
```

Comment:
Moved here, from `mnemonic`. E.g. `bip39` primitives are now hidden inside the wallet provider, not inside `mnemonic`.

Reply:
Later - extracted to `MnemonicComputer` (a bit similar in design to `AddressComputer`, `TransactionComputer`).